views/index.jade

```diff
@@ -1,128 +1,126 @@
+                                img(src='/images/business-organization.png' style='height: 40px; margin-top: 5px')
+                            .caption
+                                //- img.img-responsive(alt="create slides" src="images/create-feature-img.jpeg")    
+                                p Songwriters can share their worship songs with the Global Church on Glorify so that even more believers can worship freely.
+                                //- button.btn.btn-default Create Slides   
+                    .col-xs-12.col-sm-6.col-md-6
+                        .thumbnail(style='text-align:center')
+                            h4 Free for all to worship
+                            .circularIcon(style='margin:auto')
+                                img(src='/images/broken-link.png' style='height: 40px; margin-top: 5px')
+                            .caption
+                                //- img.img-responsive(alt="create slides" src="images/create-feature-img.jpeg")    
+                                p All songs and translations on this website are freely available as public domain or CC0 songs. Believers all over the globe can sing, share, and translate these songs.
+                                //- button.btn.btn-default Find Songs
+            div(style='text-align: center;')
+                a.btn.btn-default(href='#why') Learn more
+            hr
 
-    header#header(style="text-align: center; margin-bottom: 90px")
-        img(src='/images/glorifylogo400x100.png' style='height: 80px; margin-bottom: 50px')
-        p Find Public Domain and CC0 Worship Songs in Multiple Languages
-        #searchBox(style='width: 30em; margin: auto')
+            #why
+                h2 Why Does This Matter?
+                p.
+                    Revelation 7 says that people from every language and every part of the world will be praising God together. And yet, many worship songs that are sung in churches right now originate from English-speaking countries and are copyrighted. Churches are legally required to pay in order to perform, display, print, and even translate worship songs. And license requirements also vary depending on the country.
 
-    hr
-    .container-fluid
-    h1(style='text-align:center; margin-top: 0;') Worship Freely
-    h4(style='text-align:center; font-style:italic;') After this I looked, and behold, a great multitude that no one could number, from every nation, from all tribes and peoples and languages, standing before the throne and before the Lamb, clothed in white robes, with palm branches in their hands, and crying out with a loud voice, “Salvation belongs to our God who sits on the throne, and to the Lamb!” - Revelation 7:9-11 ESV
-    hr
+                p.
+                    We want to see the Church unleashed from the status quo. We want to see the Church united in worship across language and culture. We want to see the Church be able to worship freely. That's why we built Glorify.cc.
```

Comment:
Drop the first sentence about the status quo, or say
"We want to see the Church unleashed from the segregated status quo."